recipes/fenster/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from conan import ConanFile
+from conan.tools.apple import is_apple_os
+from conan.tools.files import copy, export_conandata_patches, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.52.0"
+
+
+class FensterConan(ConanFile):
+    name = "fenster"
+    description = "The most minimal cross-platform GUI library"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zserge/fenster"
+    topics = ("gui", "minimal")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def configure(self):
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
```

Comment:
This is a `header-only` recipe, so no actual compilation is needed, so the self.info.clear() below is enough, even if it's a c library